packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -113,14 +120,11 @@ class ControlledProgressivePaginator extends React.Component {
     const renderFirstandLastButton = totalCount > 0;
 
     const { messageId, messageAttributes } = getPageLabel(pageLabel, selectedPage, totalPages);
+    const pageDetails = (hidePageCount && pageLabel) ? <div>{pageLabel}</div> : <div>{intl.formatMessage({ id: messageId }, messageAttributes)}</div>;
```

Comment:
I think we will need to use the translation for the pure _pageLabel_ text as well to meet internationalization criteria.

Reply:
Translations will be provided by user. Since Page Label is custom string provided user we cannot translate them, Instead user should always provide translated string instead of normal string.